src/main/java/org/dataone/bookkeeper/api/Quota.java

```diff
@@ -68,6 +68,8 @@
     private String unit;
 
     /* The quota subscription id */
+    @NotEmpty
+    @NotNull
```

Comment:
See my comment in #36 - I think we need to back this change out so we can still store standard `quotas` that are associated with `products`, not `subscriptions`